src/components/panel/panel.js

```diff
@@ -306,6 +309,8 @@ angular
  * Adds a class to the panel. DO NOT use this to hide/show the panel.
  *
  * @param {string} newClass Class to be added.
+ * @param {boolean} toElement Whether or not to add the class to the panel
```

Comment:
The top ones is what is generated into the docs site. The bottom ones are the regular JSDoc for functions. In general, the ng-doc ones would have more details and examples to help devs understand how to use it. The ones below are solely for people working on the code.

Internally, every function needs to have JSDoc, including a descriptions of the method, params, and return statements. We don't have a concept of ng-doc, so we don't run into this public/private issue. The formatting is also slightly different since ngdoc needs types and has the `@description` keyword. There's also `@return` vs `@returns`.

The alternative would be to mix JSDoc and ng-doc within the actual code, but that makes it more difficult to see what is part of the public API vs internal to the panel. We decided early on that it was better to make the distinction so all public documentation is up front and grouped together.